public/sw.js

```diff
@@ -16,7 +16,7 @@ self.addEventListener('install', event => {
       ])
 
       return Promise.all(
-        ['/app.css', '/bundle.js'].map(url => {
```

Comment:
I'm sort of ambivalent about it at the moment. I removed the `bundle.js` just to reduce the number of possible things that could go wrong during development. The bundle is just around 300KB on production so I think its fine if that is the only asset that isn't currently cached right now.